java/gradlew.bat

```diff
@@ -70,6 +70,7 @@ goto fail
 
 set CLASSPATH=%APP_HOME%\gradle\wrapper\gradle-wrapper.jar
 
+
```

Comment:
Was this intentional?
It's fine if it was! Just wanted to make sure.

Reply:
pretty sure these wrappers are generated by gradle and checked in

Reply:
this is generated when you run the wrapper update comman. I just keep the generated file as is.